dev.vencord.Vesktop.metainfo.xml

```diff
@@ -0,0 +1,140 @@
+<?xml version='1.0' encoding='utf-8'?>
+<component type="desktop">
+  <!--Created with jdAppStreamEdit 7.1-->
+  <id>dev.vencord.Vesktop</id>
+  <name>Vesktop</name>
+  <summary>Vesktop is a cross platform desktop app aiming to give you a snappier Discord experience with Vencord pre-installed.</summary>
+  <developer_name>Vencord Contributors</developer_name>
+  <launchable type="desktop-id">dev.vencord.Vesktop.desktop</launchable>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0</project_license>
+  <project_group>Vencord</project_group>
+  <description>
+    <p>Vesktop is a cross platform desktop app aiming to give you a snappier Discord experience with Vencord pre-installed.</p>
```

Comment:
https://docs.flathub.org/docs/for-app-authors/appdata-guidelines/quality-guidelines#dont-repeat-the-summary